cars/car.js

```diff
@@ -1,9 +1,18 @@
 module.exports = Car
 
-function Car(mpg) {
-
+function Car({gallons, odometer, trips, mpg}){
```

Comment:
It doesn't look like these vars are being used: `{gallons, odometer, trips, mpg}`

I think the original API should be maintained `Car(mpg)` - and then variable `mpg` should be used in the constructor to set `this.mpg`